python_modules/dagster/dagster/_core/definitions/unresolved_asset_job_definition.py

```diff
@@ -48,6 +48,16 @@ class UnresolvedAssetJobDefinition(
         ],
     )
 ):
+    """Represents a job that targets an unresolved AssetSelection.
+
+    An unresolved job can't be executed directly, because it doesn't hold direct references to the
```

Comment:
Is an unresolved job a job that targets an unresolved AssetSelection? This is implied but just asking for clarity